aptos-move/aptos-gas-schedule/src/gas_schedule/transaction.rs

```diff
@@ -200,6 +200,26 @@ crate::gas_schedule::macros::define_gas_parameters!(
             { 7.. => "max_storage_fee" },
             2_0000_0000, // 2 APT
         ],
+        [
+            dependency_per_module: InternalGas,
+            { 14.. => "dependency_per_module" },
```

Comment:
Is it intentional that we bumped the LATEST_GAS_VERSION to 15 but these are added to 14?